Automattic/wp-calypso
# Add/rename badge type to Premium

Related to:
- https://github.com/Automattic/wp-calypso/issues/83153
- https://github.com/Automattic/wp-calypso/issues/84323
## Proposed Changes

* Updated the default badge text to `Premium`
* Fixed an issue that caused WooCommerce themes to show `Upgrade` when selecting a style variation.

## Testing Instructions


* Open the Calypso live link
* Go to the LiTS
* Check that the themes badge always displays `Premium` and doesn't reference `Upgrade`
* WooCommerce themes should always show the `WooCommerce` 
## Pre-merge Checklist

Before: 

https://github.com/Automattic/wp-calypso/assets/1989914/b20a10da-c3da-4dfe-9b03-ff1970d8c09e

After:

https://github.com/Automattic/wp-calypso/assets/1989914/a2771410-54cc-47a7-97e0-d2ec574c9fb0
